resources/prosody-plugins/mod_muc_lobby_rooms.lua

```diff
@@ -57,21 +62,57 @@ load_config();
 local lobby_muc_service;
 local main_muc_service;
 
--- Checks whether there is self-status 110 of the <x node
-function check_self_status(muc_x)
+-- Checks whether there is status in the <x node
+function check_status(muc_x, status)
     if not muc_x then
         return false;
     end
 
-    for status in muc_x:childtags('status') do
-        if status.attr.code == '110' then
+    for statusNode in muc_x:childtags('status') do
+        if statusNode.attr.code == status then
             return true;
         end
     end
 
     return false;
 end
 
+function broadcast_json_msg(room, from, json_msg)
+    json_msg.type = NOTIFY_JSON_MESSAGE_TYPE;
+
+    local occupant = room:get_occupant_by_real_jid(from);
+    if occupant then
+        room:broadcast_message(
+            st.message({ type = 'groupchat', from = occupant.nick })
+              :tag("json-message", {xmlns='http://jitsi.org/jitmeet'})
```

Comment:
Also here, i don't know what is the preferred quoting rule in lua, but I guess it's the same as on top